app/Helpers/SucresParser.php

```diff
@@ -139,7 +139,7 @@ public function getMentions($ret_type = self::MENTIONS_RETURN_COMPLETE)
             $target = SucresHelper::unicodeTrim(str_replace(['@', '#u:'], '', $excerpt));
             $user = User::where('name', $target)->first();
 
-            if ($user && $ret_type != self::MENTIONS_RETURN_COMPLETE) {
+            if ($user && $ret_type != self::MENTIONS_RETURN_COMPLETE && $user->getIsNotifMentionAttribute()) {
```

Comment:
Valable pour toutes les autres occurences, le `getIsNotifMentionAttribute()` s'écrit `$user->is_notif_mention`

https://laravel.com/docs/5.8/eloquent-mutators#accessors-and-mutators